virttest/libvirt_xml/vm_xml.py

```diff
@@ -607,7 +607,7 @@ def get_device_class(type_name):
         """
         return librarian.get(type_name)
 
-    def undefine(self, options=None, virsh_instance=base.virsh):
+    def undefine(self, options=None, virsh_instance=base.virsh, debug=False):
```

Comment:
we can use `**dargs` instead to have generic support.

Reply:
But this way is much simpler.

Reply:
but **dargs will make things generic and there wont be any future changes required, it will start supporting all the virsh parameters.

Reply:
But the virsh parameters are supported by 'options', and 'debug=True' is not a virsh parameter.

Reply:
I meant `readonly`, `uri`, `quiet`, `virsh_exec` (virsh binary) are some of the virsh parameters, along with that other parameters like debug, timeout, unprivileged_user will also get support with the change.

Reply:
Oh, I see. But this is not a virsh command, this is simply a support function to manipulate xml of vm: update xml of vm by executing undefine , then define.